libs/sql-parser/src/main/antlr/io/crate/sql/parser/antlr/SqlBaseParser.g4

```diff
@@ -723,7 +723,7 @@ explainOptions
    ;
 
 explainOption
-   : (ANALYZE | COSTS) booleanLiteral?
+   : (ANALYZE | COSTS | VERBOSE) booleanLiteral?
```

Comment:
Can you also add `VERBOSE` to the `nonReserved` list? Otherwise this would be a breaking change and users would need to quote `verbose` in their statements.

Reply:
Done.